studio/util.py

```diff
@@ -309,15 +321,26 @@ def download_file_from_qualified(qualified, local_path, logger=None):
             # ).communicate()
             _s3_download_dir(bucket, key, local_path, logger=logger)
         else:
-            boto3.client('s3').download_file(bucket, key, local_path)
+            s3_client = _get_active_s3_client()
+            s3_client.download_file(bucket, key, local_path)
     else:
         raise NotImplementedError
 
+def _get_active_s3_client():
+    artifact_store = model_setup.get_artifact_store()
+    if artifact_store is None \
+        or not isinstance(artifact_store, BaseArtifactStore):
+        raise NotImplementedError("Artifact store is not set up or has the wrong type")
+
+    storage_client = ((BaseArtifactStore)(artifact_store)).get_storage_client()
+    if storage_client is None:
+        raise NotImplementedError("Expected boto3 storage client for current artifact store")
+    return storage_client
```

Comment:
Extract S3 connection client where we assume to have it,
exactly the one created using studioml original configuration.